charts/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.25.0-0 < 1.32.0-0'
+  catalog.cattle.io/managed: "true"
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/rancher-version: '>= 2.10.0-0'
+  catalog.cattle.io/release-name: system-upgrade-controller
+apiVersion: v1
+appVersion: v0.14.2
```

Comment:
v0.14.2 is from October, can we use something newer?

Reply:
This matches the version used in v2.10.x release line (https://github.com/rancher/charts/blob/dev-v2.10/charts/system-upgrade-controller/105.0.1/values.yaml#L8) so we're just matching that.
Bumping SUC version is out of scope for this effort.